joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/querying/ast/CallTests.scala

```diff
@@ -22,7 +22,7 @@ class CallTests extends CSharpCode2CpgFixture {
 
       inside(writeLine.argument.l) {
         case (base: Identifier) :: (strArg: Literal) :: Nil =>
-//          base.typeFullName shouldBe "System.Console"
+          //          base.typeFullName shouldBe "System.Console"
```

Comment:
Does this base have a type yet or not? If it does then we can uncomment it

Reply:
This doesn't seem to be a test written by me, maybe `scalafmt` added this change.